### PR TITLE
Clean CIT_CMS_T2 

### PR DESCRIPTION
**Pull request by juztas**
Remove JPata from Secondary contacts.
Remove HPC - This is done using flocking.

### CODE REVIEW

**Comment by edquist** [comment]
Hi @juztas ,

Rather than removing the CMS_CIT_T2_SC_ORIGIN_01 resource, can you mark it as `Active: false` ?

Like

```
  CMS_CIT_T2_SC_ORIGIN_01:
    Active: false
    ContactLists: 
```

...

And instead of removing CIT_CMS_T2_HPC and its downtime file, can you set `Active: false` there too?

Thanks,
Carl

**Comment by juztas** [comment]
Hi, done. let me know.

File: topology/California Institute of Technology/Caltech CMS Tier2/CIT_CMS_T2.yaml [hunk 1]
```diff
@@ -502,47 +439,39 @@ Resources:
...
+     DN: /DC=org/DC=incommon/C=US/ST=CA/L=Pasadena/O=California Institute of Technology/OU=High Energy Physics/CN=stashcache-origin-01.ultralight.org
+     FQDN: stashcache-origin-01.ultralight.org
+     ID: 1009
+     Services:
+       XRootD origin server:
+         Description: StashCache origin server for user data
+     AllowedVOs:
+       - CMS
+     VOOwnership:
+       CMS: 100
```
Comment (edquist) [inline topology/California Institute of Technology/Caltech CMS Tier2/CIT_CMS_T2.yaml:474]: Somehow it looks like an extra space got added to the indentation - if you can apply this suggestion (to match the previous indentation) then the diff for the PR will more clearly show that you are just removing Joosep's contact references.

```suggestion
    Active: false
    ContactLists:
      Administrative Contact:
        Primary:
          ID: 6895284f54fb41f9daafc4b6d0a7ea42c57e31be
          Name: Justas Balcas
        Secondary:
          ID: 5e2a78ef254d5b584751f4de9cd490287301e8e7
          Name: Andres Moya Ignatov
      Resource Report Contact:
        Primary:
          ID: 6895284f54fb41f9daafc4b6d0a7ea42c57e31be
          Name: Justas Balcas
        Secondary:
          ID: 5e2a78ef254d5b584751f4de9cd490287301e8e7
          Name: Andres Moya Ignatov
      Security Contact:
        Primary:
          ID: 6895284f54fb41f9daafc4b6d0a7ea42c57e31be
          Name: Justas Balcas
        Secondary:
          ID: 5e2a78ef254d5b584751f4de9cd490287301e8e7
          Name: Andres Moya Ignatov
    DN: /DC=org/DC=incommon/C=US/ST=CA/L=Pasadena/O=California Institute of Technology/OU=High Energy Physics/CN=stashcache-origin-01.ultralight.org
    FQDN: stashcache-origin-01.ultralight.org
    ID: 1009
    Services:
      XRootD origin server:
        Description: StashCache origin server for user data
    AllowedVOs:
      - CMS
    VOOwnership:
      CMS: 100
```

**Comment by juztas** [comment]
fixed indentation

**Review by edquist — APPROVED** [review]
Great, thanks, looks good!

**Pull request merged**